### PR TITLE
Refactored all files from lower_phase_hash and lower_phase_hases to l…

### PR DESCRIPTION
**Pull request by wolfpack94**
issue #37. Modified all files from lower_phase_hash and lower_phase_hashes to lower_hash and verified that it still works using pycharm.

### CODE REVIEW

File: blockchain/network.py [hunk 1]
```diff
@@ -438,7 +438,7 @@ def phase_3_broadcast(self, block_info, phase_type):
         phase_3_msg.p2_count = verification_info['p2_count']
         phase_3_msg.business_list = verification_info['business_list']
         phase_3_msg.deploy_loc_list = verification_info['deploy_location_list']
-        phase_3_msg.lower_phase_hashes = verification_info['lower_phase_hashes']
+        phase_3_msg.lower_hash = verification_info['lower_hash']
```
Comment (micylt) [inline blockchain/network.py]: Should be lower_hashes

File: blockchain/processing.py [hunk 2]
```diff
@@ -416,22 +416,22 @@ def _execute_phase_3(self, config, phase_2_info):
                 phase_2_record = thrift_record_to_dict(phase_2_info.record)
                 phase_2_record['phase'] = phase
 
-                lower_phase_hashes = [record['signature']['hash'] for record in phase_2_records]
+                lower_hash = [record['signature']['hash'] for record in phase_2_records]
```
Comment (micylt) [inline blockchain/processing.py]: Should be lower_hashes

File: blockchain/processing.py [hunk 3]
```diff
@@ -416,22 +416,22 @@ def _execute_phase_3(self, config, phase_2_info):
...
                 # TODO: add a structure such as a tuple to pair signatory with it's appropriate hash (signatory, hash)
                 # TODO: and store that instead of lower_phase_hashes also add said structure to phase_3_msg in thrift
                 verification_info = {
-                    'lower_phase_hashes': lower_phase_hashes,
+                    'lower_hash': lower_hash,
```
Comment (micylt) [inline blockchain/processing.py]: Should be lower_hashes

File: blockchain/processing.py [hunk 4]
```diff
@@ -491,7 +491,7 @@ def _execute_phase_4(self, config, phase_3_info):
         phase_3_record = thrift_record_to_dict(phase_3_info.record)
 
         p3_verification_info = {
-            'lower_phase_hashes': phase_3_info.lower_phase_hashes,
+            'lower_hash': phase_3_info.lower_hash,
```
Comment (micylt) [inline blockchain/processing.py]: Should be lower_hashes

**Comment by wolfpack94** [comment]
done and verified.

**Comment by Voxelot** [comment]
@wolfpack94 please resolve merge conflicts.

**Pull request merged**